src/Components/Comments.jsx

```diff
@@ -0,0 +1,40 @@
+import { React, useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../utils/getCommentsByArticleId";
+import { useParams } from "react-router-dom";
+
+const Comments = () => {
+    const [isLoading, setIsLoading] = useState(true);
+    const [comments, setComments] = useState({})
+    const { article_id } = useParams();
```

Comment:
this is the correct way to get the article ID! it means the comments/article will load when a user goes directly to a particular route via the url, and doesnt have to click through to it from the list of articles so all good :D